src/cmd_line/subparsers/file.ts

```diff
@@ -9,7 +9,7 @@ export function parseEditFileCommandArgs(args: string): node.FileCommand {
     }
 
     var scanner = new Scanner(args);
-    let name = scanner.nextWord();
+    var name = scanner.nextWord();
```

Comment:
perfect. We are using `let` totally the same way. `let` always :) I'll give it a quick turnaround.